packages/shared/src/types/props/banner.ts

```diff
@@ -93,3 +76,5 @@ export const bannerSchema = z.object({
 export interface PuzzlesBannerProps extends Partial<BannerProps> {
     tracking: Tracking;
 }
+
+export { ConfigurableDesign, hexColourToString };
```

Comment:
do we need to export from this file?

Reply:
I think so! These are used elsewhere in the codebase. Or am I missing something?

Reply:
they're being imported here from another file, so I'm not sure why this file needs to export them

Reply:
Oh ha I see what you mean, let me investigate. I must have moved things around and these got left behind.

Reply:
Fixed, thanks!